README.md

```diff
@@ -25,6 +25,22 @@ _Why_: this encourages developers to write lines that do less, and extract varia
 
 We don’t have an 80 char hard limit because sometimes it’s more readable to let a line be 85 characters rather than break it up, but in general you should aim for 80 characters or less.
 
+#### Use `String.prototype.includes` to check for substrings
+
+_Why_: clearer intention than using `indexOf`, and is part of the ES6 specification.
```

Comment:
Thanks a bunch for wrapping this up - I hadn't got my head round the final conclusion yet!
Could we word it here the same as you did below, for extra niceness? `its intent is much clearer than using...`